libraries/lps/include/mcrl2/lps/lps_rewriter_type.h

```diff
@@ -74,6 +81,8 @@ std::string description(const lps_rewriter_type type)
       return "for one point rule quantifier elimination";
     case condition_one_point :
       return "simplify summands using equalities appearing in condition";
+    case dataspec_prune:
+      return "for removing unused data specifications";
```

Comment:
Perhaps more accurately to remove unused equations (and sorts if that is included?).